src/test_application/activescenes/scenarios_vehicletest.cpp

```diff
@@ -111,6 +127,8 @@ struct VehicleTestData
 
     osp::ResId m_prefab;
```

Comment:
Check warning on line 107 in src/test_application/activescenes/scenarios_vehicletest.cpp

Code scanning

Variable 'testapp::scenes::VehicleTestData::m_prefab' is uninitialized. Always initialize a member variable (type.6).

Warning
Variable 'testapp::scenes::VehicleTestData::m_prefab' is uninitialized. Always initialize a member variable (type.6).
Show more details